keras/src/saving/saving_lib.py

```diff
@@ -205,12 +235,68 @@ def _save_model_to_fileobj(model, fileobj, weights_format):
                 zf.write(weights_file_path, weights_file_path.name)
 
 
+def _upload_model_to_hf(model, hf_path, weights_format):
+    if hf_path.startswith("hf://"):
+        hf_path = hf_path[5:]
+    if hf_path.count("/") > 1:
+        raise ValueError(
+            "Invalid `hf_path` argument: expected `namespace/model_name`"
```

Comment:
Probably slightly better to keep the original user path around and raise with that. Rather than this truncated version (caller might wonder where the prefix went.

expected `"hf://namespace/model_name"`. Received hf_path=...

Reply:
Good idea :+1: Addressed in https://github.com/keras-team/keras/pull/19854/commits/4dcf9db6ab88e306fc3059a82702298200d2ab03.